src/helpers/transaction.ts

```diff
@@ -195,7 +194,7 @@ export const makeTxConfigFromSignedTx = (
         ? getStoreAccount(accounts)(decodedTx.from as TAddress, networkDetected.id) ||
           oldTxConfig.senderAccount
         : oldTxConfig.senderAccount,
-    gasPrice: gasPriceToBase(parseInt(decodedTx.gasPrice, 10)).toString(),
+    gasPrice: gasPriceToBase(decodedTx.gasPrice).toString(),
```

Comment:
Can this be a BigNumber or hex-string rather than a decimal string?

Reply:
This is for formatting and showing only, so should not be an issue. Also it would take a significant amount of time to check that the hex is then handled in every UI properly.